src/components/Nav/Navigation.tsx

```diff
@@ -37,7 +37,9 @@ const istioConfigRx = /\/namespaces\/(.*)\/istio\/(.*)/g;
 type PropsType = {
   location: any;
   authenticated: boolean;
+  navCollapsed: boolean;
```

Comment:
Please remove it (navCollapsed) from the [StateType](https://github.com/kiali/kiali-ui/pull/497/files#diff-a642fcb9370181978f2dc88c65a5e69eR47) if no longer used.

Reply:
I forgot it !!! Great thanks @josejulio  changed !!!